optuna/samplers/_tpe/parzen_estimator.py

```diff
@@ -21,7 +21,6 @@
 
 
 class _ParzenEstimatorParameters(NamedTuple):
-    consider_prior: bool
     prior_weight: float | None
```

Comment:
How about remove `None` from the `prior_weight` type?

```suggestion
    prior_weight: float
```